apps/guide/content/docs/voice/index.mdx

```diff
@@ -11,61 +11,53 @@ title: Installation
 To add voice functionality to your discord.js bot, you will need the `@discordjs/voice` package, as well as one of the encryption packages listed below. For example:
 
 ```sh tab="npm"
-npm install @discordjs/voice libsodium-wrappers
+npm install @discordjs/voice
 ```
 
 ```sh tab="yarn"
-yarn add @discordjs/voice libsodium-wrappers
+yarn add @discordjs/voice
 ```
 
 ```sh tab="pnpm"
-pnpm add @discordjs/voice libsodium-wrappers
+pnpm add @discordjs/voice
 ```
 
 ```sh tab="bun"
-bun add @discordjs/voice libsodium-wrappers
+bun add @discordjs/voice
 ```
 
 After this, you'll be able to play Ogg and WebM Opus files without any other dependencies. If you want to play audio from other sources, or want to improve performance, consider installing some of the extra dependencies listed below.
 
-<Callout>
-	This guide assumes you have installed at least one additional dependency – FFmpeg. More information on this can be
```

Comment:
we still assume this, considering the AudioPlayer and AudioResource examples all use .mp3 files.